src/zoid/auth/component.jsx

```diff
@@ -62,93 +72,119 @@ export function getAuthComponent() : ZoidComponent<AuthPropsType> {
                     required:   false
                 },
 
-                clientID: {
-                    type:       'string',
-                    value:      () => getClientID()
+                client_id: {
+                    type:               'string',
+                    queryParam:         true,
+                    value:      () =>   getClientID(),
+                    required:           false
```

Comment:
I recommend keeping this required. The `client-id` is required for the JS SDK to load. So it will always be there.

```suggestion
```

Reply:
done